ui/src/app/tags/views/TagUpdate/TagUpdateFormFields/TagUpdateFormFields.tsx

```diff
@@ -65,14 +65,15 @@ export const TagUpdateFormFields = ({ id }: Props): JSX.Element | null => {
           <KernelOptionsField id={id} />
         </Col>
         <Col size={6}>
-          {/* // TODO: Add the link to the docs:
-          // https://github.com/canonical-web-and-design/app-tribe/issues/748 */}
           {!tag.definition ? (
             <Definition label={Label.Definition}>
               <span className="p-form-help-text">
                 This is a manual tag. Definitions cannot be added to manual
                 tags. To learn more about this, check our{" "}
-                <a href="#todo">XPath documentation</a>.
+                <a href="https://maas.io/docs/how-to-work-with-tags#heading--xpath-expressions">
```

Comment:
Same here:
- Id missing from the document
- Can we make the link the entire sentence for a11y and provide a clearer user expectation when clicking this link?
- Should these links open in a new tab?

Reply:
Couldn't agree more on your points, but I didn't even look at the context as I was making quick link updates for @huwshimi - we need this before the 3.2 release today.

Reply:
I think the main part here which shouldn't be contentious is to improve the link text.

Reply:
When it comes to opening docs in a new tab - I'm pretty sure we decided against it as a general rule but can't find a source at the moment. Maybe @amylily1011 could help?